client/state/invites/actions.js

```diff
@@ -196,57 +199,63 @@ export function createAccount( userData, invite ) {
 }
 
 export function generateInviteLinks( siteId ) {
-	return ( dispatch ) => {
-		wpcom
-			.undocumented()
-			.site( siteId )
-			.generateInviteLinks()
-			.then( () => {
-				dispatch( requestSiteInvites( siteId ) );
-			} );
+	return async ( dispatch ) => {
+		await wpcom.req.post( {
+			path: `/sites/${ siteId }/invites/links/generate`,
+			apiNamespace: 'wpcom/v2',
+		} );
+		dispatch( requestSiteInvites( siteId ) );
 	};
 }
 
 export function disableInviteLinks( siteId ) {
-	return ( dispatch ) => {
-		wpcom
-			.undocumented()
-			.site( siteId )
-			.disableInviteLinks()
-			.then( () => {
-				dispatch( requestSiteInvites( siteId ) );
-			} );
+	return async ( dispatch ) => {
+		await wpcom.req.post( {
+			path: `/sites/${ siteId }/invites/links/disable`,
+			apiNamespace: 'wpcom/v2',
+		} );
+		dispatch( requestSiteInvites( siteId ) );
 	};
 }
 
 export function acceptInvite( invite ) {
 	return async ( dispatch ) => {
-		const data = await wpcom.undocumented().acceptInvite( invite );
 		try {
+			const data = await wpcom.req.get(
+				{
+					path: `/sites/${ invite.site.ID }/invites/${ invite.inviteKey }/accept`,
+					apiVersion: '1.2',
```

Comment:
@jsnajdr 👋 Was working on something related, and I noticed that this recent change makes this API call hit the old `v1.1` endpoint, instead of `v1.2`, Placing the `apiVersion` param in the second object (with `activate` and `include_domain_only`) fixes it.

Reply:
Nice find @annemirasol ! Fixing in #53660.